save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt

```diff
@@ -314,58 +290,98 @@ val createVulnerabilityView: VFC = VFC {
                                 }
 
                                 div {
-                                    className = ClassName("col-12 px-2 mt-3 d-flex align-items-center justify-content-between")
-
-                                    div {
-                                        className = ClassName("form-label mb-0 text-left")
-                                        +"Affected projects:"
+                                    className = ClassName("col-12 px-2 mt-2 text-left")
+                                    label {
+                                        className = ClassName("form-label mb-0")
+                                        +"Other users"
+                                    }
+                                    addUserComponent {
+                                        idsToSkip = vulnerability.participants.mapNotNull { it.id }.toSet()
+                                        onUserAdd = { newUser ->
+                                            setVulnerability { vuln -> vuln.copy(participants = vuln.participants.plus(newUser)) }
+                                        }
                                     }
+
                                     div {
-                                        className = ClassName("")
-                                        buttonBuilder(faPlus, isOutline = true, classes = "btn-sm rounded-circle") {
-                                            projectWindowOpenness.openWindow()
+                                        vulnerability.participants.forEach { user ->
+                                            div {
+                                                className = ClassName("mt-2 d-flex align-items-center")
+                                                div {
+                                                    className = ClassName("col-2 d-flex justify-content-center align-items-center")
+                                                    img {
+                                                        className = ClassName("img-fluid")
+                                                        src = user.avatar ?: "img/undraw_profile.svg"
```

Comment:
I would reduce the size of the avatars by 1.5 or 2 times

Reply:
makes sense

Reply:
done